spec/data/terraform-config.yaml

```diff
@@ -12,9 +12,4 @@
 # limitations under the License.
 
 --- !ruby/object:Provider::Terraform::Config
-objects: !ruby/object:Api::Resource::HashArray
-  BackendBucket:
-    id: "{{name}}"
-    ignore: ["id"]
-    validation:
-      name: "^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$"
\ No newline at end of file
+overrides: !ruby/object:Provider::ResourceOverrides
```

Comment:
Is that supposed to be in here?

Reply:
A few of these files seem unrelated.

Reply:
Nope, ignore me!  I think I just missed that the files below this were under `spec/`.